python/dgl/distributed/partition.py

```diff
@@ -177,8 +276,9 @@ def partition_graph(g, graph_name, num_parts, out_path, num_hops=1, part_method=
     * `part_method` is the method used to assign nodes to partitions.
       Currently, it supports "random" and "metis".
     * `num_parts` is the number of partitions.
-    * `halo_hops` is the number of HALO nodes we want to include in a partition.
+    * `halo_hops` is the number of hops of nodes we include in a partition as HALO nodes.
```

Comment:
Use either
```
``halo_hops``
```
or
```
:attr:`halo_hops`
```
for monospace fonts.  Single backquotes won't render correctly.

Please scan for other places as well as there are a lot of misuses..